lib/internal/quic/core.js

```diff
@@ -2688,13 +2762,13 @@ class QuicStream extends Duplex {
     // TODO(@jasnell): Implement this later
   }
 
-  get pending() {
-    // The id is set in the kSetHandle function
-    return this[kInternalState].id === undefined;
-  }
-
-  get aborted() {
-    return this[kInternalState].aborted;
+  get detached() {
+    // The QuicStream is detached if it is yet destroyed
+    // but the underlying handle is undefined. While in
+    // detached mode, the QuicStream may still have
+    // data pending in the read queue, but writes will
+    // not be permitted.
+    return this[kHandle] === undefined;
```

Comment:
Is this needed? Shouldn't `readable`/`writable` suffice?

Reply:
No, detached means that the underlying native handle is not set. It's possible we don't need this, but I want to keep it in for now. We can pull it back out later if it ends up not being necessary. It's been useful for debugging for the time being.